index.html

```diff
@@ -180,5 +181,47 @@ <h3>Skills</h3>
         </div>
       </section>
     </section>
+
```

Comment:
- [ ] Inside your cards you used classed starting with uppercase letter like: `Feat-text`, `Feature`. I would recommend changing them into lowercase `feat-text`, `feature` to make your HTML & CSS good 🍀